### PR TITLE
Launch Pad Lamp v1.0 from Github

### PR DESCRIPTION
**Pull request by droric**
My Launch Pad Lamp mod.   My KSP forum username is Trolllception.

### CODE REVIEW

**Comment by droric** [comment]
Not sure why this is failing.  I am able to run netkan against the file and it generates a .CKAN file and installs.

**Comment by linuxgurugamer** [comment]
CKAN identifiers must consist only of letters, numbers, and dashes, and must start with a letter or number.

**Comment by linuxgurugamer** [comment]
Remove the spaces from the identifier

**Comment by droric** [comment]
Hmm again I am at a loss here.

File: NetKAN/LaunchPadLamp.netkan [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "spec_version" : "v1.0",
+    "identifier"   : "LaunchPadLamp",
+    "$kref"        : "#/ckan/github/droric/Launch-Pad-Lamp",
+    "license"      : "CC-BY-NC-SA-3.0",
+    "ksp_version"  : "1.3.0",
+    "resources": {
+        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/164677-13-launch-pad-lamp/"
+    }
```
Comment (politas) [inline NetKAN/LaunchPadLamp.netkan]: You'll need to add an install clause:

    "install": [
        {
            "find":"Launch Pad Lamp",
            "install_to": "GameData"
        }
    ]

File: NetKAN/LaunchPadLamp.netkan [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "spec_version" : "v1.0",
```
Comment (politas) [inline NetKAN/LaunchPadLamp.netkan]: "spec_version" : "v1.4",

Comment (politas) [reply NetKAN/LaunchPadLamp.netkan]: spec_version "v1.0" is not actually valid. I'm not sure why it isn't complaining, but for this mod it should be one of the following lines:

    "spec_version" : "v1.4",
    "spec_version" : 1,

File: NetKAN/LaunchPadLamp.netkan (hunk 1 above)
Comment (politas) [reply NetKAN/LaunchPadLamp.netkan]: I gather you've re-done the zipfile so it has a matching folder to the mod identifier. That works fine.

**Comment by politas** [comment]
Looks great now. Merging with thanks!

**Pull request merged**